spec.md

```diff
@@ -549,6 +549,28 @@ Analog<32> ; 32-bit analog type
 Analog     ; analog type with inferred width
 ```
 
+### Enumeration Types
+
+Enumerations are structural disjoint union types.  An enumeration has a number
+of variants, each with a type.  The different variants are specified with tags.
+The variant types of an enumeration must all be passive and cannot contain
+analog or probe types.
+
+In the following example, the first variant has the tag `a` with type `UInt<8>`,
```

Comment:
It's a pain, but consider adding the `{.firrtl}` to these, so they are highlighted (consistently).